index.js

```diff
@@ -6,6 +6,7 @@ var path = require("path");
 var assign = require("object-assign");
 var fastreplace = require('./lib/fastreplace');
 var findNestedRequires = require('./lib/findNestedRequires');
+var minify = require('html-minifier').minify;
```

Comment:
Yes, this is an added dependency. I feel like it's probably better to add a known and tested library to do this sort of thing than writing my own regular expressions which will surely be buggy.